.gitignore

```diff
@@ -138,3 +138,9 @@ portfolio/portfolio.sln
 portfolio/portfolio
 portfolio/node_modules/
 portfolio/src_burger/
+
+# inflation
+inflation/chrome-data/
+inflation/dataset/chrome-data/
+inflation-resources/dataset/chrome-data
+inflation-resources/tests/dataset/chrome-data/
```

Comment:
you can use this instead: **/chrome-data (this will ignore things anywhere)

check out this: https://www.atlassian.com/git/tutorials/saving-changes/gitignore